### PR TITLE
chore: add CircleCI config

### PR DESCRIPTION
**Pull request by skuruppu**

### CODE REVIEW

File: .circleci/config.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build_and_test:
+    docker:
+    - {image: 'circleci/golang:1.13'}
```
Comment (hengfengli) [inline .circleci/config.yaml:5]: Change to `- image: circleci/golang:1.13`.

Comment (hengfengli) [reply .circleci/config.yaml:5]: Take a look at this example: https://github.com/cloudspannerecosystem/harbourbridge/blob/master/.circleci/config.yml

**Review by hengfengli — COMMENTED** [review]
I think you can refer to https://github.com/cloudspannerecosystem/harbourbridge/blob/master/.circleci/config.yml, which covers all formats you need here.

**Comment by skuruppu** [comment]
@hengfengli thanks so much. I copied it from https://app.circleci.com/pipelines/github/cloudspannerecosystem/harbourbridge/361/config but it turns out that config is different to what's committed. Now I've copied it from the harbourbridge repo as you suggested.

**Review by hengfengli — APPROVED** [review]
LGTM.

**Pull request merged**